independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/extensions/ExtensionManager.java

```diff
@@ -4,6 +4,7 @@
 import io.quarkus.bootstrap.model.AppArtifactKey;
 import io.quarkus.dependencies.Extension;
 import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.registry.InstallPlan;
```

Comment:
@gastaldi are you sure you want to introduce a coupling here?

Reply:
It's just a DTO, I'll probably move it to devtools-common then. Thanks!